### ISSUE TITLE
Could not mount /data and unable to find crypto footer.

### ISSUE THREAD

**Issue by kid1412621** [TeamWin/android_device_oneplus_dumpling#1]
Android 8.0 (H2OS beta2)
TWRP version:3.2.1-1

**Comment by friederbluemle** [comment]
@kid1412621 Yeah, TWRP 3.2.1-1 is not compatible with the Open Beta builds of OxygenOS (Android 8.0). We'll probably have to wait until the final version is out and TWRP has been fixed.

**Comment by kid1412621** [comment]
@friederbluemle Thanks for reply, I installed third-party build version for temporary solution.
So , how about the support schedule as far as you know ?

**Comment by friederbluemle** [comment]
Interesting. Could you share which third party build version you're using? Unfortunately I don't know the schedule either.

**Comment by kid1412621** [comment]
@friederbluemle I tried this one ,twrp-3.2.1-0-universal-codeworkx-dumpling.img ,learnt from XDA. 
Actually it is a fork of this project.Here is the link:https://github.com/codeworkx/android_device_oneplus_dumpling/tree/android-8.0

**Comment by Martinusbe** [comment]
codeworkx pushed everything to twrp & omni gerrit

**Comment by kid1412621** [comment]
@Martinusbe Push to TWRP?

**Comment by friederbluemle** [comment]
@kid1412621 Very cool, thanks for the link. I'll try this later tonight.

**Comment by friederbluemle** [comment]
twrp-3.2.1-0-universal-codeworkx-dumpling.img works perfectly for me. Now using latest Open Beta 3 (Android 8.0.0) for my OnePlus 5T.
